coverage/clover.xml

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are these supposed to be included in the repo or can they be added to the `.gitignore`?